components/back-link/src/index.js

```diff
@@ -55,8 +55,8 @@ const Anchor = styled('button')({
   },
 });
 
-const BackLink = ({ children, className, goBack }) => (
-  <Anchor className={className} onClick={goBack}>{children}</Anchor>
+const BackLink = ({ children, goBack, ...props }) => (
+  <Anchor onClick={goBack} {...props}>{children}</Anchor>
```

Comment:
was wondering if we should place these things around the other way, i.e. `<Anchor {...props} onClick={goBack}>` but doing it this way around means that we can support using `onClick` as a prop which would override `goBack`, which fits nicely with the notion that we should rename `goBack` to `onClick` - i.e. we're already supporting that :grin: